source/extensions/tracers/zipkin/span_context.h

```diff
@@ -20,20 +20,24 @@ class SpanContext {
   /**
    * Default constructor. Creates an empty context.
    */
-  SpanContext() : trace_id_(0), id_(0), parent_id_(0), is_initialized_(false), sampled_(false) {}
+  SpanContext()
+      : trace_id_(0), trace_id_high_(0), id_(0), parent_id_(0), is_initialized_(false),
+        sampled_(false) {}
 
   /**
    * Constructor that creates a context object from the supplied trace, span and
    * parent ids, and sampled flag.
    *
-   * @param trace_id The trace id.
+   * @param trace_id The low 64 bits of the trace id.
+   * @param trace_id_high The high 64 bits of the trace id.
```

Comment:
nit: usually we put the high bits before the low bits when ordering

ex trace_id_high then trace_id